live-hls.js

```diff
@@ -345,6 +344,12 @@ const extractResourceWindow = function(mfest, duration, event, streamtype) {
   debuglog('event start ' + event.start);
 
   event.discomod = 0;
+
+  if (header.Discontinuity.value) {
```

Comment:
Is this safe?  Should it be `(header.Discontinuity && header.Discontinuity.value)`?

Reply:
It should be, the properties are created when the header object is created 
https://github.com/videojs/murphy/blob/master/live-hls.js#L74